utils/overage_team.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import {OverActiveUserLimits} from './constants';
```

Comment:
I cannot for the life of me find where this is coming from. Doesn't seem to be defined in that file? I'm assuming they're decimal numbers representing the % thresholds?

Reply:
It's available in the [mm-47645-show-license-count-activated-users](https://github.com/mattermost/mattermost-webapp/tree/mm-47645-show-license-count-activated-users) branch. Probably hasn't been merged into the main branch yet.